"TypeError: Cannot read property 'replace' of undefined" on running the cli `query` command with several SQL statements

**Issue type:**

<!--
     Have a question?
     Check the "Support" Documentation on the best places to ask questions!
     ---
     https://github.com/typeorm/typeorm/blob/master/docs/support.md
-->

[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

On running the cli `query` command with several SQL statements all wrapped with double quotes I receive the following error from the cli-highlight:

```
> ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli --config ormconfig-migrations.ts "query" "SELECT * FROM folder;
SELECT * FROM sub_folder;"

Running query: SELECT * FROM folder;
SELECT * FROM sub_folder;
Query has been executed. Result: 
Error during query execution:
TypeError: Cannot read property 'replace' of undefined
    at escape (/usr/src/app/node_modules/highlight.js/lib/highlight.js:71:18)
    at Object.highlight (/usr/src/app/node_modules/highlight.js/lib/highlight.js:799:18)
    at Object.highlight (/usr/src/app/node_modules/cli-highlight/src/index.ts:97:21)
    at Function.PlatformTools.highlightJson (/usr/src/app/src/platform/PlatformTools.ts:221:16)
    at Object.<anonymous> (/usr/src/app/src/commands/QueryCommand.ts:55:39)
    at step (/usr/src/app/node_modules/typeorm/node_modules/tslib/tslib.js:141:27)
    at Object.next (/usr/src/app/node_modules/typeorm/node_modules/tslib/tslib.js:122:57)
    at fulfilled (/usr/src/app/node_modules/typeorm/node_modules/tslib/tslib.js:112:62)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sindy@1.0.0 typeorm: `ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli --config ormconfig-migrations.ts "query" "SELECT * FROM folder;
npm ERR! SELECT * FROM sub_folder;"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sindy@1.0.0 typeorm script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-24T13_13_49_857Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sindy@1.0.0 rls:on: `bash src/policies/enable-rls.sh`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sindy@1.0.0 rls:on script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-24T13_13_49_873Z-debug.log

```

--- a/src/commands/QueryCommand.ts
+++ b/src/commands/QueryCommand.ts
@@ -55,8 +55,13 @@ export class QueryCommand implements yargs.CommandModule {
             queryRunner = connection.createQueryRunner();
             console.log(chalk.green("Running query: ") + PlatformTools.highlightSql(args._[1]));
             const queryResult = await queryRunner.query(args._[1]);
-            console.log(chalk.green("Query has been executed. Result: "));
-            console.log(PlatformTools.highlightJson(JSON.stringify(queryResult, undefined, 2)));
+
+            if (typeof queryResult === "undefined") {
+                console.log(chalk.green("Query has been executed. No result was returned."));
+            } else {
+                console.log(chalk.green("Query has been executed. Result: "));
+                console.log(PlatformTools.highlightJson(JSON.stringify(queryResult, undefined, 2)));
+            }
 
             await queryRunner.release();
             await connection.close();

